app/build.gradle

```diff
@@ -134,6 +134,9 @@ dependencies {
     implementation Deps.androidx_constraintlayout
     implementation Deps.androidx_preference
     implementation Deps.google_material
+    implementation ('me.dm7.barcodescanner:zxing:1.9.8') {
```

Comment:
What does this library offer over zxing directly?

If this related to this GitHub repository then I wonder how trustworthy this library is:
https://github.com/dm77/barcodescanner

Reply:
basically saves us from writing all this and a bit more: https://github.com/dm77/barcodescanner/blob/ffeaeaf8a0fc2ff247056eb731cb0500df7c2dc4/zxing/src/main/java/me/dm7/barcodescanner/zxing/ZXingScannerView.java 

We want to use this in various products as well, so maybe we can pull this into some kind of a component, but it is tricky due to the fxa dependency 

cc @csadilek